snippets/python/object-memory-size.py

```diff
@@ -0,0 +1,12 @@
+# Find size of an object in memory.
+
+import sys
+
+
+def object_size(obj):
+    """Return size of object in bytes. This will only work reliably for builtin types."""
+    return sys.getsizeof(obj)
+
+
+object_size(2)
+object_size([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
```

Comment:
This only returns the size of the list itself, not of the content of the list. For instance, it doesn't matter if you put in 10x a 2GB numpy array, it will not care about that, or 10x very long strings.
So, I think this is a bit misleading because it gives me the impression it will return the aggregate size.